app/helpers/curation_concerns/file_set_helper.rb

```diff
@@ -7,9 +7,9 @@ def parent_path(parent)
     end
   end
 
-  def media_display(file_set, locals = {})
-    render media_display_partial(file_set),
-           locals.merge(file_set: file_set)
+  def media_display(presenter, locals = {})
```

Comment:
@cam156 Ah. So perhaps the method is now in the wrong helper module? Like, should there be a media_display_helper module? That could be a follow-up refactor? Maybe add a TODO (or REVIEW) item in the comments?